content/en/watchdog/faulty_cloud_saas_api_detection.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Automatic Faulty Cloud & SaaS API Detection
+further_reading:
+# - link: "https://www.datadoghq.com/blog/<URL FROM THE BLOGPOST>/"
+#   tag: "Blog"
+#   text: "Stay ahead of service disruptions with Watchdog Cloud and API Outage Detection"
+- link: "watchdog/faulty_deployment_detection/"
+  tag: "Documentation"
+  text: "Learn about Watchdog Faulty Service Deployment Detections"
+---
+
+## Overview
+
+Automatic Faulty Cloud & SaaS API Detection finds third-party providers (payment gateways, cloud providers, etc) having issues within minutes, reducing mean time to detection (MTTD). Watchdog continuously monitors for elevated error rates in requests, using APM telemetry, to external providers—such as AWS, OpenAI, Slack, Stripe, and more—in order to detect service degradation as soon as it occurs. This proactive detection gives you a head start in identifying and mitigating issues before they escalate, significantly reducing time spent on root cause analysis and improving response times.
+
+When Watchdog identifies that an external provider you are using is faulty, it adds which services are impacted by the problem and the extent of the disruption, allowing you to differentiate between external and internal issues quickly. Datadog also provides direct links to the provider’s status page and support channels, making it easy to escalate and inform them of the issues.
```

Comment:
```suggestion
When Watchdog identifies that an external provider you are using is faulty, it flags the services impacted by the problem and the extent of the disruption. This allows you to differentiate between external and internal issues. Datadog also provides direct links to the provider's status page and support channels, so you can reach out to them as needed.
```

Reply:
It's not clear to me what Watchdog "adding" services means, so I've updated this to `flag` in the above suggestion. Feel free to make any additional updates if there's a better way to express this.